mapquest_geocoder.go

```diff
@@ -58,7 +63,11 @@ func (g *MapQuestGeocoder) Request(url string) ([]byte, error) {
 // if one occurs during the geocoding request.
 func (g *MapQuestGeocoder) Geocode(query string) (*Point, error) {
 	url_safe_query := url.QueryEscape(query)
-	data, err := g.Request(fmt.Sprintf("search.php?q=%s&format=json", url_safe_query))
+	key := ""
+	if (MapquestAPIKey != "") {
+		key = "key="+MapquestAPIKey+"&"
+	}
+	data, err := g.Request(fmt.Sprintf("search.php?%sq=%s&format=json",key, url_safe_query ))
```

Comment:
The Nominatim search API docs don't mention using a key [here](http://open.mapquestapi.com/nominatim/), but after glancing at the other implementations that do use a key, It seems as though you may want to have this format string be:

`search.php?key=%s&q=%s&format=json`